src/renderer/entities/price/model/__tests__/price-provider-model.test.ts

```diff
@@ -74,4 +74,13 @@ describe('entities/price/model/price-provider-model', () => {
     await allSettled(priceProviderModel.events.assetsPricesRequested, { scope, params: { includeRates: false } });
     expect(scope.getState(priceProviderModel.$assetsPrices)).toEqual(prices);
   });
+
+  test('should update asset prices on currency change', async () => {
```

Comment:
Let's stick to actual `store` / `events` names
```suggestion
  test('should update $assetPrices when currencyChanged', async () => {
```